.vortex/tooling/playground/.env

```
@@ -0,0 +1,39 @@
+PLAYGROUND_WITH_EQUALS=value=with=equals
+
+# Variable that can be overridden by .env.local
+PLAYGROUND_OVERRIDE=original_value
+
+# Comment line - should be ignored
+# PLAYGROUND_COMMENT=this_should_not_load
+
+# Empty lines are okay
+
+# Multi-word value without quotes
+PLAYGROUND_MULTIWORD=this has spaces and should work
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)